### PR TITLE
Issue no 577 bug fix baseline crackmapexec

### PR DESCRIPTION
**Pull request by niloy1997**

### CODE REVIEW

**Comment by connorwjordan** [comment]
Need to run prettier on the Crackmapexec.tsx file as it has failed the style check.

File: src/components/Crackmapexec/Crackmapexec.tsx [hunk 1]
```diff
@@ -6,38 +6,78 @@ import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
...
+ * The Crackmapexec tool helps in penetration testing for Windows/Active Directory environments.
+ * 
+ * Steps:
+ * 1. Enter the target IP address.
+ * 2. Enter the username.
+ * 3. Enter the password.
+ * 4. Click "Start Searching" to execute the Crackmapexec tool.
+ * 5. Review the output results in the Output block.
+ * 
+ * Further information can be found at: https://www.kali.org/tools/crackmapexec/
```
Comment (connorwjordan) [inline src/components/Crackmapexec/Crackmapexec.tsx:40]: This should be located in a sourceLink constant.

File: src-tauri/exploits/cve-2020-1472.py
Comment (connorwjordan) [inline src-tauri/exploits/cve-2020-1472.py:1]: This should not be deleted.

File: src-tauri/exploits/log4j-shell-poc/Exploit.class
Comment (connorwjordan) [inline src-tauri/exploits/log4j-shell-poc/Exploit.class:1]: This should not be deleted.

File: src-tauri/exploits/log4j-shell-poc/Exploit.java
Comment (connorwjordan) [inline src-tauri/exploits/log4j-shell-poc/Exploit.java:1]: This should not be deleted.

File: src-tauri/exploits/zerologon_tester.py
Comment (connorwjordan) [inline src-tauri/exploits/zerologon_tester.py:1]: This should not be deleted.

File: src/components/Crackmapexec/Crackmapexec.tsx [hunk 2]
```diff
@@ -6,38 +6,78 @@ import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
 import { UserGuide } from "../UserGuide/UserGuide";
```
Comment (connorwjordan) [inline src/components/Crackmapexec/Crackmapexec.tsx:6]: UserGuide is deprecated and should be removed.
```suggestion
```

File: src/components/Crackmapexec/Crackmapexec.tsx [hunk 3]
```diff
@@ -6,38 +6,78 @@ import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
...
+    "Crackmapexec is a post-exploitation tool that helps in penetration testing for Windows/Active Directory environments. " +
+    "It is capable of enumerating users, shares, and computers, executing commands, and dumping credentials from NTDS.dit. " +
+    "It automates various tasks to aid in testing the security of a network.\n\n" +
+    "Steps:\n" +
+    "1. Enter the target IP address.\n" +
+    "2. Enter the username.\n" +
+    "3. Enter the password.\n" +
+    "4. Click 'Start Searching' to execute the Crackmapexec tool.\n" +
+    "5. Review the output results in the Output block.\n\n" +
+    "Further information can be found at: https://www.kali.org/tools/crackmapexec/";
```
Comment (connorwjordan) [inline src/components/Crackmapexec/Crackmapexec.tsx:52]: The source link should be its own constant that contains the link to the source document.

**Review by connorwjordan — COMMENTED** [review]
There is no dependencies constant.
There is no useEffect or RenderComponent in use, import all necessary libraries for this.
A lot of constants names have been changed away from the convention in the Airbaseng.tsx reference, eg. loading and setLoading changed to isLoading and setIsLoading. https://github.com/Hardhat-Enterprises/Deakin-Detonator-Toolkit/blob/main/src/components/AirbaseNG/AirbaseNG.tsx

**Comment by niloy1997** [comment]
Here, i have made the changes

**Pull request closed without merging**